CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Enhancement (`@grafana/faro-web-sdk`): Auto extend a session if the Faro receiver indicates that a
   session is invalid (#591).
+- Enhancement (`@grafana/faro-web-sdk`): track `web vital` attribution (#595).
```

Comment:
I think this is a feature not just an enhancement :)

Reply:
I fixed it here https://github.com/grafana/faro-web-sdk/pull/595/commits/c8102aa8f445868a0ac368acb77da7fc5b75cf1b

> Hey @vitebo yes please rename it! When I wrote the suggestion for the name it was a typo that the "s" was missing.
Great catch!

I took advantage and renamed the option here too: https://github.com/grafana/faro-web-sdk/pull/595/commits/7a73c271b2e8f72067937b26f734b7c67cc21552